apps/fabric-website-resources/test/FocusZone.test.dom.ts

```diff
@@ -0,0 +1,42 @@
+import * as puppeteer from 'puppeteer';
+
+describe('FocusZone', () => {
+  let browser: any;
+  let page: any;
+
+  beforeAll(async () => {
+    browser = await puppeteer.launch({ headless: true });
+    page = await browser.newPage();
+    await page.goto('http://localhost:4322/#examples/focuszone');
+  });
+
+  afterAll(async () => {
+    await browser.close();
+  });
+
+  it('can use arrows vertically', async () => {
+    // Act
+    // 'Nesting FocusZons in list rows'
+    const focusZone = (await page.$$('.ExampleCard-example'))[1];
+    await focusZone.focus();
+
+    // Assert
+    expect((await getActiveElementText(page)).indexOf('Item-1')).toBeTruthy();
```

Comment:
this one is confusing; getting the text, and then getting the index of 'Item-1'... -1 and 1 are truthy values... is there something I'm missing here?